test/.default_config

```diff
@@ -1,7 +1,7 @@
-tests.build-image-version=latest
+tests.build-image-version=1.0
```

Comment:
raaah the bump-version script still have some problems ... this should not be modified ...

Reply:
I'll fix it as soon as we got the OperatorHub PR merged, so we can run the CI just once.